website/docs/r/mssql_managed_instance_security_alert_policy.html.markdown

```diff
@@ -0,0 +1,263 @@
+    }
+  }
+}
+
+resource "azurerm_subnet_network_security_group_association" "example" {
+  subnet_id                 = azurerm_subnet.example.id
+  network_security_group_id = azurerm_network_security_group.example.id
+}
+
+resource "azurerm_route_table" "example" {
+  name                          = "routetable-mi"
+  location                      = azurerm_resource_group.example.location
+  resource_group_name           = azurerm_resource_group.example.name
+  disable_bgp_route_propagation = false
+  depends_on = [
+    azurerm_subnet.example,
+  ]
+}
+
+resource "azurerm_subnet_route_table_association" "example" {
+  subnet_id      = azurerm_subnet.example.id
+  route_table_id = azurerm_route_table.example.id
+}
+
+resource "azurerm_mssql_managed_instance" "example" {
+  name                = "managedsqlinstance"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+
+  license_type       = "BasePrice"
+  sku_name           = "GP_Gen5"
+  storage_size_in_gb = 32
+  subnet_id          = azurerm_subnet.example.id
+  vcores             = 4
+
+  administrator_login          = "mradministrator"
+  administrator_login_password = "thisIsDog11"
+
+  depends_on = [
+    azurerm_subnet_network_security_group_association.example,
+    azurerm_subnet_route_table_association.example,
+  ]
+}
+
+resource "azurerm_mssql_managed_instance_security_alert_policy" "example" {
+  resource_group_name        = azurerm_resource_group.example.name
+  managed_instance_name      = azurerm_mssql_managed_instance.example.name
+  state                      = "Enabled"
+  storage_endpoint           = azurerm_storage_account.example.primary_blob_endpoint
+  storage_account_access_key = azurerm_storage_account.example.primary_access_key
+  disabled_alerts = [
+    "Sql_Injection",
+    "Data_Exfiltration"
+  ]
+  retention_days = 20
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `resource_group_name` - (Required) The name of the resource group that contains the MS SQL Managed Instance. Changing this forces a new resource to be created.
+
+* `managed_instance_name` - (Required) Specifies the name of the MS SQL Managed Instance. Changing this forces a new resource to be created.
+
+* `state` - (Required) Specifies the state of the policy, whether it is enabled or disabled or a policy has not been applied yet on the specific database. Possible values are `Disabled`, `Enabled`.
+
+* `disabled_alerts` - (Optional) Specifies an array of alerts that are disabled. Possible values are `Sql_Injection`, `Sql_Injection_Vulnerability`, `Access_Anomaly`, `Data_Exfiltration`, `Unsafe_Action` and `Brute_Force`.
+
+* `email_account_admins` - (Optional) Boolean flag which specifies if the alert is sent to the account administrators or not. Defaults to `false`.
```

Comment:
```suggestion
* `email_account_admins_enabled` - (Optional) Boolean flag which specifies if the alert is sent to the account administrators or not. Defaults to `false`.
```

Reply:
@katbyte  thanks for your feedback. Fixed.